### ISSUE TITLE
Design for User Interface of Community Attribute Table

### ISSUE THREAD

**Issue by kbwagner** [openopps/OpenOppsTasks#107]
Create the front end for entering information on the community.  

Provide list of fields
Attach Tracy's wireframes.

**Comment by humancompanion** [comment]
https://github.com/openopps/openopps-platform/issues/2550

**Comment by humancompanion** [comment]
@kbwagner - Has this happened? Is it ready for review yet? If not, Alex could work on mocking this up. However, we would really prefer the Program Office pick Communities as a Lab research topic before we start building them out. We have a potential pilot partner at GSA. Regardless, we can do an MVP design of this if you have it slated to be done soon-ish.

**Comment by kbwagner** [comment]
@humancompanion no, it hasn't happened yet.  We have a backlog ticket but it's pretty far down so won't likely be done for a few more sprints.  This is one that we agreed development could use the design system and work the ticket and then have design review.  However, if Alex has time to work on it I'm sure development would appreciate it

**Comment by kbwagner** [comment]
@abudz19 some additional information: 
We really need to revisit this table at some point in full and see if all of the attributes are really being used now that we know more about communities.

I think as part of each deployment, we will ask for the community attribute information and our team will enter it for them.  But, they will need to be able to make changes to fields that aren’t system changing.  I’ll have to look but Brice had a list in the actual ticket of things we shouldn’t let them change because it will effect how the system operates – like target audience.

I see it being a page somewhere on the Admin side…maybe only system admin can  create a new one but once created the community admin can edit.  

Here’s the actual Macon ticket: : https://github.com/openopps/openopps-platform/issues/2837

Brody has requested some additional fields to be entered on the table: https://github.com/openopps/openopps-platform/issues/3082

Here’s the original Create community ticket so you can see the fields: https://github.com/openopps/openopps-platform/issues/2550

**Comment by abudz19** [comment]
@humancompanion Wanted to check this in to make sure it's going in a good direction: https://opm.invisionapp.com/d/main#/console/17049184/353420496/preview
Still need to add Brody's additional fields.

**Comment by abudz19** [comment]
FYR @humancompanion @kbwagner @Piglady64 
Edit community - fully wired: https://opm.invisionapp.com/d/main#/projects/prototypes/17049184

**Comment by abudz19** [comment]
Removed community administrator. Community administrator = community manager, per conversation with Katrina.

**Comment by kbwagner** [comment]
@abudz19 - this ticket is written.  I think this one can be closed.

**Comment by abudz19** [comment]
https://opm.invisionapp.com/d/main#/console/17049184/353420497/preview @kbwagner what do you think about this instead?

**Comment by kbwagner** [comment]
@abudz19 I like it.  I wonder if the community name should still be first since it's kind of the key field...but get how you have it ordered.

Looks like we lost the community admin name and email though.  and skills.

Thanks!

**Comment by abudz19** [comment]
@kbwagner also changed it from "edit community" to "manage community" if you missed that on the first and third pages.

**Comment by abudz19** [comment]
@kbwagner is this okay to be closed?

**Comment by kbwagner** [comment]
@abudz19 yes ok to close
